source/platforms/github/GitHubUtils.ts

```diff
@@ -194,7 +194,7 @@ export const createOrAddLabel = (pr: GitHubPRDSL | undefined, api: GitHub) => as
   d("Checking for existing labels")
   let label: any = null
   try {
-    const existingLabels = await api.issues.listLabelsForRepo({ owner: config.owner, repo: config.repo })
+    const existingLabels = await api.paginate('GET /repos/:owner/:repo/labels', { owner: config.owner, repo: config.repo })
```

Comment:
I haven't been able to test this, but checking the documentation, seems like the right way of doing this 😅 
https://github.com/octokit/rest.js/tree/8373934ed21eb69c8dbacc3e06e1dc1566a6a7d8#pagination

Reply:
That's a lot of labels! So this doesn't change the response shape?

Reply:
I know 😅 But I use a repo that gets automatic dependency bumps and each of them generates a new label 😓 

Looking at what I think is the source code, `listLabelsForRepo` seems to be doing this call internally, so the response should be the same
https://github.com/octokit/plugin-rest-endpoint-methods.js/blob/cb9d3e408a245e8b49971cf61bf76dcab4742669/src/generated/endpoints.ts#L458

Reply:
The CI fails are probably correct in this case I think

Reply:
You're totally right @orta, the build was failing because the response changes slightly and `.data` is not needed (`existingLabels.find` instead of `existingLabels.data.find`).

The build still failing, but only for Node 10.16, because "No output has been received in the last 10m0s". I've tried it a couple of times. Do you think that's related to my changes, or it's some Travis issue?